crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -265,6 +265,55 @@ impl CairoHintProcessor<'_> {
                     .stop_mock_call(contract_address, &function_name);
                 Ok(())
             }
+            "start_spoof" => {
+                fn make_option<T>(is_value_set: bool, value: T) -> Option<T> {
+                    if is_value_set {
+                        Option::Some(value)
+                    } else {
+                        Option::None
+                    }
+                };
+
+                let contract_address = ContractAddress(PatriciaKey::try_from(StarkFelt::new(
+                    inputs[0].clone().to_be_bytes(),
+                )?)?);
+
+                let version = make_option(inputs[1].clone().is_one(), inputs[2].clone());
+                let account_contract_address =
+                    make_option(inputs[3].clone().is_one(), inputs[4].clone());
+                let max_fee = make_option(inputs[5].clone().is_one(), inputs[6].clone());
+                let transaction_hash = make_option(inputs[7].clone().is_one(), inputs[8].clone());
+                let chain_id = make_option(inputs[9].clone().is_one(), inputs[10].clone());
+                let nonce = make_option(inputs[11].clone().is_one(), inputs[12].clone());
+
+                let signature_len = inputs[14]
+                    .to_usize()
+                    .expect("Missing signature len in inputs");
```

Comment:
I think this error is not from a missing signature, but from type conversion failure.